.circleci/config.yml

```diff
@@ -37,28 +38,11 @@ _refs:
       git config credential.helper 'cache --timeout=120'
       git config user.email "$GH_EMAIL"
       git config user.name "Release Bot"
-  gus-create-change-mgmt: &gus-create-change-mgmt
-    name: 'Gus create change case management'
+  gus-prepare-environment-variables: &gus-prepare-environment-variables
+    name: 'Prepare environment variables for GUS Change Case Management'
     command: |
-      if [ "$GUS_SKIP" == "false" ]; then
-        sudo npm install -g @salesforce/change-case-management
-        export GUS_BUILD="$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"
-        echo "Gus build is ${GUS_BUILD}"
-        sfchangecase create -i ${GUS_TEMPLATE_ID} --release offcore.tooling.${GUS_BUILD} --location ${REPO_LOCATION}
-        sfchangecase check --release offcore.tooling.${GUS_BUILD} --location ${REPO_LOCATION}
-      else
-        echo "Gus change case management was skipped because GUS_SKIP is true"
-      fi
-  gus-close-change-mgmt: &gus-close-change-mgmt
-    name: 'Gus close change case management'
-    command: |
-      if [ "$GUS_SKIP" == "false" ]; then
-        export GUS_BUILD="$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"
-        echo "Gus build is ${GUS_BUILD}"
-        sfchangecase update --location ${REPO_LOCATION} --release offcore.tooling.${GUS_BUILD} --status 'Closed - Deploy Successful'
-      else
-        echo "Gus change case management was skipped because GUS_SKIP is true"
-      fi
+      export GUS_BUILD="$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"
+      echo "export GUS_RELEASE=offcore.tooling.$GUS_BUILD" >> $BASH_ENV
```

Comment:
This was mentioned at https://circleci.com/docs/2.0/env-vars/#using-parameters-and-bash-environment as a way to share ENV VARIABLES between steps. This needs to be verified with a sample run.